src/site/content/en/blog/quintoandar/index.md

```diff
@@ -0,0 +1,226 @@
+- QuintoAndar has over 30 [React](https://reactjs.org) web applications. Delivering updates to these applications and maintaining them in accordance to best practices is an arduous task.
+
+## Approach
+
+We began a performance optimization project of the condominium content hub to improve its user experience, as these improvements could lead to conversion gains, better SEO, and better usability. This initiative was also a fitting opportunity to improve the developer experience as well.
+
+### Migrating to Next.js
+
+The new version of the condominium page was implemented with [Next.js](http://nextjs.org/). Being largely independent from other parts of the app, the condominium content hub seemed like a good candidate for trying out a new framework. We would be able to understand the magnitude of migration efforts and evaluate how its features could help without affecting the other React apps in QuintoAndar.
+
+A hard requirement was to ensure pages remained crawlable by search engines. Next.js meets this requirement by supporting server-side rendering out-of-the-box, and removes the need for a custom setup. The documentation makes it much easier to share knowledge on how to do tasks such as [data fetching](https://nextjs.org/docs/basic-features/data-fetching) on the server and onboard new developers. Server-side rendering is also known to [improve performance](https://developers.google.com/web/updates/2019/02/rendering-on-the-web#server-rendering) metrics such as [First Contentful Paint](/fcp/) (FCP).
+
+The framework provides other performance-friendly features such as automatic [code splitting](/reduce-javascript-payloads-with-code-splitting/) and [prefetching](/link-prefetch/). Even though the existing structure already provided such features, the additional work required from developers stalled their adoption — for example, code splitting at page or component-level had to be done manually.
+
+### Optimizing JavaScript resources
+
+The first step was to [remove unused code](/remove-unused-code/). We looked at the [Webpack Bundle Analyzer](https://github.com/webpack-contrib/webpack-bundle-analyzer) reports, which shows the contents of each JS bundle, and carefully reviewed all third-party scripts. As a result, we were able to clean up some tracking libraries that were not used in this specific page.
+
+Our team went further and evaluated the performance cost of existing features. For instance, the "like" button required quite a lot of JS to work. However, in the condominium page, less than 0.5% of the users interacted with the button, which is available and used more frequently in other parts of our app. After a discussion involving both Engineering and Product, we decided to remove this feature.
+
+<figure>
+  {% Video src="video/jL3OLOhcWUQDnR4XjewLBx4e3PC3/2uEL2YxbuGGlmJDlvW9F.mp4", autoplay=true, muted=true, playsinline=true, loop=true, controls=true %}
```

Comment:
Is there a way we can center the `<video>` within `<figure>`?

Reply:
Yep. Fixed this and all other containers.